### DISCUSSION TITLE
Urgenza

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#135] [Richieste di aiuto] [answered]
Buonasera, vorrei sapere una cosa:una volta che si conclude lo scrutinio finale,ovvero il secondo quadrimestre, come si fa a spostare gli alunni promossi ex.della 1A in 2A e così via?Inoltre,quando si conclude l'anno scolastico, è possibile non perdere i dati scolastici dell'anno precedente prima di passare all'anno successivo?Intendo,è possibile sempre consultare gli anni passati?

**Comment by trinko** [comment, marked as answer]
Al termine dell'anno scolastico non c'è alcun automatismo per passare all'anno successivo.
Il registro non mantiene un archivio degli anni passati, anche se si possono mantenere installazioni differenti per anno, come faceva anche LAMPSCHOOL.


Attualmente sono operative solo queste funzioni:
- possibilità di importare i dati dal database dell'anno precedente, limitatamente agli utenti, alle circolari, agli esiti degli scrutini
- dopo l'importazione, nella nuova installazione sono visibili:
   -  ad alunni/genitori gli esiti del solo anno precedente;
   -  ai docenti: gli esiti finali dei soli alunni delle proprie cattedre;
   - alla segreteria, gli esiti del solo anno precedente.

**Comment by sempliciniluigi89** [comment]
Quando usavo LAMPSCHOOL ,questo permetteva a fine scrutinio di secondo
quadrimestre di spostare tutti gli alunni nelle classi successive ,ma va
bene



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
01/22/22,
08:25:01 PM

Il giorno sab 22 gen 2022 alle ore 19:58 Antonello Dessì <
***@***.***> ha scritto:
